Week-5/Work & HW/index/war.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>War</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <style>
+    #alert {
+      font-size: 300px;
+    }
+    </style>
+  </head>
+  <body>
+    <div id="top">
+    </div>
+    <h1>Your Hand</h1>
+    <div id="div">
+    </div>
+    <button id="war">War!</button>
+    <div id="div2">
+      <h1>Other Person</h1>
+    </div>
+
+    <script>
+      let cards = [];
+      let cardsimages = [];
+      let cards2 = [];
+      let cards2images = [];
+      let checkNum = (num, array) => {
+        if (num.includes("2") || num.includes("3") || num.includes("4") || num.includes("5") || num.includes("6") || num.includes("7") || num.includes("8") || num.includes("9") || num.includes("10")) {
+          num = parseInt(num);
+          array.push(num);
+        } else {
+          array.push(num);
+        }
+      }
+      let check =  (array) => {
```

Comment:
`check` is a bad name for this function. Why not `convertArrayOfCardsToValues` or `computeCardValues` or something?